components/input/input.css

```diff
@@ -0,0 +1,135 @@
+@import url("../../style-guide/compiled-style-guide.css");
+body {
+  background-color: var(--bg-white);
+  font-family: "Exo", sans-serif;
+}
+
+.txt-highlight {
+  color: var(--primary-color);
+}
+.component-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem;
+}
+/*CSS FOR ENCLOSED INPUT VARIANTS*/
+.gentle-form-group,
+.gentle-input-group {
+  display: flex;
+  flex-direction: column;
+  color: var(--secondary-color);
+}
+.gentle-input-label {
+  font-size: 1rem;
+  margin: 0.5rem;
+}
+.gentle-input {
+  font-size: 1.2rem;
+  padding: 0.5rem 0.75rem;
+  outline: none;
```

Comment:
Any reason to use outline:none here?

Reply:
So basically if we click on the input section without this property a yellow border shows up whenever we are entering sometthing into the input so just to avoid that yellow border to mix up with my styling I am giving this declaration to it